client/reader/search-stream/index.jsx

```diff
@@ -235,9 +235,11 @@ class SearchStream extends Component {
 							delayTimeout={ 500 }
 							placeholder={ searchPlaceholderText } />
 					</CompactCard>
-					<p className="search-stream__blank-suggestions">
-						{ this.props.translate( 'Suggestions: {{suggestions /}}.', { components: { suggestions: sugList } } ) }
-					</p>
+					{ this.state.suggestions &&
```

Comment:
I'm happy to change it, but what would the benefit be in this case?

`pickedSuggestions` (which is put into `this.state.suggestions`) is always `null` if the language has no suggestions.